src/views/Login/Login.tsx

```diff
@@ -1,38 +1,63 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import { createStructuredSelector } from 'reselect';
 
-import { requestToken } from './actions';
+import { requestToken, changeLogin, changePassword } from './actions';
+import { selectUserLogin, selectPassword } from './selectors';
 
 interface MappedProps {
-    requestToken: (username: string, password: string) => Promise<any>;
+    login: string;
+    password: string;
 }
 
-type Props = MappedProps;
+interface MappedActions {
+    requestToken: (username: string, password: string) => Promise<any>;
```

Comment:
I am not understanding this string. "=> Promise<any>"

Reply:
Basically, this is an async function and it returns the Promise.
Promise<any> means that the promise will have any response, when it will resolve.
You can type the response of async function using Promise<CustomResponseInterface>